README.md

```diff
@@ -1,4 +1,22 @@
 displayer-multiselect-suggest
```

Comment:
If we are documenting the parameters for edit mode we should also add in a separate commit (misc: update documentation) the documentation of the pre-existing parameters (the doc is on extensions)